src/environments/environment.ts

```diff
@@ -1,7 +1,8 @@
 export const environment = {
     production: false,
 
-    apiUrl: 'https://demo.todos.gateway.api.kros.wtf',
+    apiUrl: 'http://localhost:9000',
+    // apiUrl: 'https://demo.todos.gateway.api.kros.wtf',
```

Comment:
toto ked sa nasadi tak nepojde treba localhost vymazat a demo api odkomentovat

Reply:
Neni na to ten druhý file environmnet.prod.ts ?

Reply:
to hej ale ak si niekto forkne tak potrebuje pristup pretoze backend nema chapes a tu moze byt testovacia api a na prod je produkcna nemusia byt rovnake v kazdom pripade tam nema byt localhost